### ISSUE TITLE
"Wiser Heat" app download geographically restricted to UK (and Ireland?) - tips welcome

### ISSUE THREAD

**Issue by james-cook** [asantaga/wiserHomeAssistantPlatform#217]
Hallo wiserHAP people,

It's perhaps an edge case but I live mostly outside the UK/Ireland and I am seeking to remotely smart manage the heating in an old family place in Ireland.
I had been thinking of using a raspi3 to run HA with these wiser components and access remotely using port forwarding.

Drayton claim: "Download The Wiser App for your smart phone by searching "Wiser Heat" in the app store. This will give you 24/7 access to your heating from anywhere in the world. ". Sounds good.

But... I have just noticed that the "Wiser Heat" app in the playstore is geographically restricted to the UK(!) 
I know I can change my location in the playstore (once a year) but this would be a real pain.

* Can the drayton wiser system be setup without the app - e.g. using only this project? 
* Does anyone have experience installing the "Wiser Heat" app from an alternative reliable apk source?
* I assume people have the app working in Ireland (that the playstore means UK and Ireland when it says "UK")

It seems strange that Drayton would restrict the app download in this way.

All thoughts welcome :)

**Comment by rlewis187** [comment]
You would need to set the heating up with the app first to get the API key for the integration to work. 

Looks like you can download the latest version here:
https://apkpure.com/wiser-home/com.schneider_electric.WiserHeat
At least I can in England. 

Are you sure there are no restrictions on operating the heat hub outside of the UK? I can't see any good reason why there would be, once set up with HA, this essentially wouldn't matter anyway.

UK officially includes Northern Ireland, but not Ireland (I assume), Brexit may have complicated the whole thing somewhat.

**Comment by asantaga** [comment]
I concur with rlewis, you need the official app to set it up and then you could usee this integration to do the rest.

**Comment by james-cook** [comment]
Thanks for your responses :)

**UK and Ireland?** -  well the documentation at https://schneider-electric.zendesk.com/hc/en-gb/articles/360014454077-I-can-t-find-the-Wiser-Heat-app-in-the-App-Store suggests the UK only(!) but this must be incorrect(?) Screwfix sell drayton wiser here in Dublin.

@rlewis187 "Are you sure there are no restrictions on operating the heat hub outside of the UK? I can't see any good reason why there would be, once set up with HA, this essentially wouldn't matter anyway." 
I'm not sure about **operation**. I don't think drayton would geo-block operation/registration/updates by IP address, but who knows - if they are already blocking the app download.
For clarity, tis issue is just about **downloading** the Wiser Heat/Home app from google playstore in the first place.


Thanks for the link to apkpure - this is an option but carries "some" security risk. I have asked drayton if they could just provide a link to their apk directly.

**Comment by asantaga** [comment]
@james-cook You dont need a rooted device to install the apps from the APK but if you have a rooted device then you can download the apps, using google play, and then save the APKs  to dropbox etc. If your device is not rooted then , its possible , but a hellava lot more difficult and google dont make it easy..

Just did a quick google and there are websites which do it for you. e.g. https://www.wikihow.com/Download-an-APK-File-from-the-Google-Play-Store

I prefer to save my own APKs, I know they havent been tainted..

My recommendation is

1. Find another android device, setup google+play in the UK, setup Wiser and then use the integration
2. Use the various ways to get a APK (third party sites, tools, the link above) or a rooted device

Thats it really...

**Comment by james-cook** [comment]
@asantaga 
Thanks for all of your suggestions - really appreciated :)

I am trying to avoid it but setting up a 2nd cheap "UK" android device does seem like the only real/cleanest way to go.

Am I right in thinking that potentially  I'd only need this 2nd phone to use the official wiser app for install/setup? API key etc.
After install/setup the hub itself would update automatically anyway.
I/family wouldn't need the official android app at all(?) - we could just use the port forwarded HA interface from then on.
This would save us all from having to carry extra phones just to access the heating remotely.
Or is there an app/wiserHAP interaction that is still needed somewhere.

**Comment by rlewis187** [comment]
Any way you can use an Android emulator such as BlueStacks (https://www.bluestacks.com/) rather than a completely separate device?

**Comment by rlewis187** [comment]
> I/family wouldn't need the official android app at all(?) - we could just use the port forwarded HA interface from then on.

I find the HA Wiser integration far quicker to respond and easier to use than the Official Wiser App, especially remotely.

**Comment by james-cook** [comment]
> Any way you can use an Android emulator such as BlueStacks (https://www.bluestacks.com/) rather than a completely separate device?

Wow, I hadn't thought of emulation! May save me from buying a 2nd android device at all!

**Comment by asantaga** [comment]
Yes, rlewis187 is right! bluestacks emulation would work a treat!

In my household we primarily use the App and HA is used for data capture and fancy automations.. Everyone is different

**Comment by m01** [comment]
If I were in your shoes I'd reach out to Wiser support and Screwfix, since you're probably not the only one who'd end up running into this issue if Screwfix are selling it in Ireland. There may be valid reasons why they're trying to restrict it to the UK.

**Comment by asantaga** [comment]
> If I were in your shoes I'd reach out to Wiser support and Screwfix, since you're probably not the only one who'd end up running into this issue if Screwfix are selling it in Ireland. There may be valid reasons why they're trying to restrict it to the UK.

They probably just pushed it to google play in the uk play store, didnt click the international market. I do agree worth raising it with wiser support, costs them nothing to publish to international store..

Im going to close this issue, I think we've covered all bases but feel free to reopen

**Comment by james-cook** [comment]
@m01 - Yep, I opened a ticket with them on Thursday night (13th Jan). Yet to hear back.

@asantaga - Closing seems fine to me :) Thanks to everyone for their tips. 
It does seems that drayton/schneider/eberle have published the app to UK/Ireland only without thinking it completely through. The advertising says, "control your heating from all over the world", it does not add  "but only if your are a UK user".

Just to restate the current situation:

1.  Wiser Home or Wiser Heat (whatever the current name is) is not visible in Google Playstore if your google user location is not UK (and probably Ireland too). This makes it impossible to register the hardware after installation.
2. Drayton's own zendesk support site suggests the app is [not available outside the UK](https://schneider-electric.zendesk.com/hc/en-gb/articles/360014454077-I-can-t-find-the-Wiser-Heat-app-in-the-App-Store) (i.e. only visible to users with "UK" location)
3. All Drayton Wiser hub hardware (e.g. kits 1, 2, 3 etc.) is available to buy in Ireland (e.g. screwfix.ie etc.) and people are reporting their installations e.g. on the "boards.ie" website. Irish heating installations are VERY similar to UK installations so it makes sense.
4. so point 3. seems to suggest the users with location "Ireland" have the app in their google playstore (I can't verify this)

I'll post here again if I get a reply from Drayton

**Comment by james-cook** [comment]
FWIW I tried bluestacks but couldn't find a way to connect to the hub via WiFi. 

I didn't speed to long on this, so maybe it is possible.

In the end what does work is registering another Gmail user in playstore and in my case registering an Irish debit card for this user. YMMV.
To be clear I am also in Ireland so the IP address is clearly in Ireland too.

Now I have 2 users in playstore and each are shown different apps in the store.
I was able to download the Wiser app and then install and commission the system without a problem.
